v0/destinations/am/data/AmplitudeRevenueConfig.json

```diff
@@ -9,11 +9,11 @@
   "context.app.build": "app_version",
   "context.device.type": "platform",
   "context.locale": "language",
-  "properties.productId": "productId",
+  "properties.product_Id": "productId",
```

Comment:
I think it should be all lower `product_id`. 

Ref: https://docs.rudderstack.com/rudderstack-api-spec/rudderstack-ecommerce-events-specification/ordering#product-viewed

Reply:
Updated